components/Workspaces/TrackedRepoWizard/PasteReposStep.tsx

```diff
@@ -12,7 +12,14 @@ export const PasteReposStep = ({ onBulkAddRepos }: PasteReposStepProps) => {
     // split each line into a trimmed string and filter out any empty lines
     const repos = pastedInput
       .split(pastedInput.includes(",") ? "," : "\n")
-      .map((line) => line.trim())
+      .map((line) => {
+        line.trim();
+        if (line.startsWith("https://github.com")) {
```

Comment:
Do you mean to get rid of `https://` on paste?

Reply:
Just trying to get clarificatio as well @jpmcb.

It's already trimmed, see line 16 above, and is only stripped of `https://github.com/` if that's what the entry starts with.

Reply:
Sorry for not being clear: If I paste in

```
github.com/spf13/cobra
```

This wasn't working.

Note: there's no leading protocol (https://) to the URL and it wouldn't go down the path since the `if` block depended on the string starting with a literal `https://github.com`.

But looks like now, with Zeu's latest changes, this was moved down into the regex and the `https://` leading bit is optional 👏🏼